homeassistant/components/sonos/media_player.py

```diff
@@ -300,13 +300,13 @@ def is_volume_muted(self) -> bool | None:
         """Return true if volume is muted."""
         return self.speaker.muted
 
-    @property  # type: ignore[misc]
-    def shuffle(self) -> str | None:
+    @property
+    def shuffle(self) -> str | None:  # type: ignore[override]
         """Shuffling state."""
         shuffle: str = PLAY_MODES[self.media.play_mode][0]
```

Comment:
```suggestion
    def shuffle(self) -> bool | None:
        """Shuffling state."""
        shuffle = PLAY_MODES[self.media.play_mode][0]
```
`shuffle` should, and already does, return an optional `bool`. The `str` type hint is just wrong.